### PR TITLE
Implemented Windows.ApplicationModel.Calls.PhoneCallManager

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): #1114

## PR Type
What kind of change does this PR introduce?
 - Feature 

## What is the current behavior?
`PhoneCallManager` is not supported.

## What is the new behavior?
`PhoneCallManager` has support:

iOS - everything except `ShowPhoneCallSettingsUI` and `RequestStoreAsync`
Android - everything except `RequestStoreAsync`
WASM - only `ShowPhoneCallUI` is supported

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information
Needed to add `Windows.ApplicationModel.Calls.CallsPhoneContract` to SyncGenerator

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UWPSyncGenerator/Generator.cs [hunk 1]
```diff
@@ -65,12 +65,13 @@ public virtual void Build(string basePath, string baseName, string sourceAssembl
...
 
-			var origins = from externalRedfs in _referenceCompilation.ExternalReferences
+			var origins = (from externalRedfs in _referenceCompilation.ExternalReferences
 						  where Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Windows.Foundation")
 						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Windows.Phone.PhoneContract")
+						  || Path.GetFileNameWithoutExtension(externalRedfs.Display).StartsWith("Windows.ApplicationModel.Calls.CallsPhoneContract")
 						  let asm = _referenceCompilation.GetAssemblyOrModuleSymbol(externalRedfs) as IAssemblySymbol
 						  where asm != null
-						  select asm;
+						  select asm).ToArray();
```
Comment (davidjohnoliver) [inline src/Uno.UWPSyncGenerator/Generator.cs]: Does this need to be included?

File: src/Uno.UWP/ApplicationModel/Calls/PhoneCallManager.Android.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+namespace Windows.ApplicationModel.Calls
+{
+	public partial class PhoneCallManager
+	{
+		private static readonly TelephonyManager _telephonyManager;
+
+		static PhoneCallManager()
+		{
+			_telephonyManager = (TelephonyManager)ContextHelper.Current
```
Comment (davidjohnoliver) [inline src/Uno.UWP/ApplicationModel/Calls/PhoneCallManager.Android.cs]: ContextHelper.Current may be null if this is called too early. Might be good to throw an informative exception in that case.

**Comment by ghuntley** [comment]
@all-contributors please add @MartinZikmund for code and tests

**Comment by MartinZikmund** [comment]
Thank you, @ghuntley :-) !

**Pull request merged**